Frameworks/CoreText/CTRun.mm

```diff
@@ -256,11 +261,15 @@ void CTRunDraw(CTRunRef run, CGContextRef ctx, CFRange textRange) {
         range.length = [string length];
     }
 
-    int numGlyphs = curRun->_characters.size();
+    int numGlyphs = curRun->_glyphAdvances.size();
     WORD* glyphs = (WORD*)IwMalloc(sizeof(WORD) * numGlyphs);
 
+    WORD* characters = (WORD*)IwMalloc(sizeof(WORD) * numGlyphs);
```

Comment:
Yes it needs to do. Good catch

---
In reply to: [77732196](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77732196) [](ancestors = 77732196)